### ISSUE TITLE
Remove /vendor

### ISSUE THREAD

**Issue by kris-nova** [kris-nova/kubicorn#264]
Once we start offering releases we need to remove the vendor directory, and the documentation on `go get` as our program will no longer be `go get`able

**Comment by prateekgogia** [comment]
Why do we want to remove the vendor, any reasons except CGO failures ? 
Removing causes pain to get started with the project, any new comer will have to go through the pain of fetching third party packages, before compiling.

**Comment by robjloranger** [comment]
I think it's just to remove the vendor directory? That way the repo is lighter and people can update the dependencies from the manifest after cloning.

**Comment by xmudrii** [comment]
@robjloranger Actually, it's a downside that repo is lighter. Removing `/vendor` would "disable" `go get` as dependencies are missing. This is a good thing actually, as it's forcing users to use releases instead of development versions. Development versions could be buggy, while released one are not that buggy.

But there's a downside.. In order to get ready to develop stuff, you need to run `dep ensure` so you can install. I'm not sure what's the current situation, but as much as I love `dep`, it was still very slow. If an contributor wants to do a quick change, it would need to lose like 30 minutes doing `dep ensure` so it can test it.

Basically, that's the biggest reason why I'm against removing `/vendor`. If `dep` gets fast enough, that could be a good idea, but for now, I think it's a very bad and demotivating one.

**Comment by robjloranger** [comment]
@xmudrii `dep ensure` is pretty quick on my machine, but it is a bit new and I know too well not everyone has a quick development machine.

So is this something to hold off on?
